packages/babel-helper-hoist-variables/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "main": "lib/index.js",
   "dependencies": {
+    "@babel/traverse": "workspace:^7.12.9",
```

Comment:
I really dislike this, but this helper is used only by plugins so there is already a `@babel/traverse` dependency somewhere.

I wish `yarn`/`npm` supported `typeDependencies`, only downloaded for TS users.

Reply:
We can re-export types exports of `@babel/traverse` in `@babel/core` or `@babel/types`. I think `NodePath` is the only heavily used type in `@babel/traverse`.

Reply:
+1 in @babe/types

Reply:
Let's re-export this from `@babel/core`, since it already depends on `@babel/traverse` (`@babel/types` doesn't).

I'll add a TODO so that we don't forget to update it after converting `@babel/core`.